### PR TITLE
Update configuration file conversion script

### PR DESCRIPTION
**Pull request by fhagemann**
Following the changes in #168, the configuration conversion script had to be updated.
While testing it, I found a bug in the `Cone` read-in (90806ee) that would lead to misinterpretations of`VaryingCylinder`.

To update config files, the user has to load `<SSD>/test/update_config_file.jl` 
(which defines a function `update_config_file()`) 
and pass the filename to that function `update_config_file` to update it to v0.6.0.

### CODE REVIEW

File: src/ConstructiveSolidGeometry/VolumePrimitives/Cone.jl [hunk 1]
```diff
@@ -604,6 +604,39 @@ function Geometry(::Type{T}, t::Type{Cone}, dict::AbstractDict, input_units::Nam
...
+        end
+        dict["r"]["top"] = if c.r[2] isa Tuple{<:Real} 
+            c.r[2][1]
+        elseif isnothing(c.r[2])
+            0
+        else 
+            OrderedDict{String, Any}("from" => c.r[2][1], "to" => c.r[2][2])
+        end
+    end
+    if !isnothing(c.φ) dict["phi"] = OrderedDict("from" => string(c.φ[1])*"rad", "to" => string(c.φ[2])*"rad") end
```
Comment (fhagemann) [inline src/ConstructiveSolidGeometry/VolumePrimitives/Cone.jl:633]: Non-2π phi-intervals will always be saved in units of `rad` with the respective unit attached.

File: src/ConstructiveSolidGeometry/Transformations.jl [hunk 2]
```diff
@@ -25,3 +25,17 @@ function combine_transformations(current::Transformations, new::Transformations)
     (rotation = r, translation = t)
 end
 
+
+function Dictionary(m::SMatrix{3,3,T,9}) where {T}
+    dict = OrderedDict{String, Any}()
+    mat = RotXYZ(m)
+    if mat.theta1 == 0 && mat.theta2 == 0
+        if mat.theta3 != 0
+            dict["Z"] = string(mat.theta3)*"rad"
```
Comment (fhagemann) [inline src/ConstructiveSolidGeometry/Transformations.jl:34]: Angles of rotation matrices around `z` will be saved in units of `rad`.

**Review by fhagemann — COMMENTED** [review]
One thing: `Dictionary`, right now, does not know about units. Thus, all angles are saved in units of "rad". There might be a more elegant work-around that this..

**Review by lmh91 — APPROVED** [review]
Looks fine to me :+1:

**Comment by fhagemann** [comment]
Do we deal with units later (#68)?

**Comment by lmh91** [comment]
> Do we deal with units later (#68)?

Sorry, I don't understand your question. #68 is about the IO of calculated fields, e.g. the ElectricPotential.
Not configuration files. Am I missing something?

**Comment by fhagemann** [comment]
> > Do we deal with units later (#68)?
> 
> Sorry, I don't understand your question. #68 is about the IO of calculated fields, e.g. the ElectricPotential.
> Not configuration files. Am I missing something?

Oh, I thought #68 was also about how to pass units to the methods that calculate stuff (that you can give ticks of plots also in `mm` and not in unitless SI numbers) and how to deal with units in general. We should add a description to that issue.. :sweat_smile:

**Comment by lmh91** [comment]
I think #68 is about some other IO of the simulated fields in addition to HDF5. 

But, yes, regarding plotting this would indeed be nice. But it does not belong here (this PR) in my opinion. 
We can later add some (optional) field to the configuration file like
```yaml
plotunits:
  lenghts: mm
  ...
```
or derive it from the general `unit` field.

**Comment by fhagemann** [comment]
Ok, then let's merge this and care about units later.

**Pull request merged**